### PR TITLE
Fix gemspec and increase version

### PR DESCRIPTION
**Pull request by karlbaker02**
### Context
Fixing gemspec to remove `ruby` when getting the version number, and increasing version number to account for the previous two commits.

### Changes proposed in this pull request
Fix gemspec and update version number.

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-0.2.0
\ No newline at end of file
+0.3.0
```
Comment (gidsg) [inline .version]: can we call this `0.2.1`? I don't think we need a major version bump for this.

Comment (karlbaker02) [reply .version]: It's a minor bump. I think we should because this is actually a breaking change.

Comment (karlbaker02) [reply .version]: The `this` referencing the previous two commits where the change was actually made.

Comment (gidsg) [reply .version]: Please update the README, so the example still works.

Comment (arnau) [reply .version]: Now that you mention that, "0.3.0" is not a major change, it is a minor according to semver of course (MAJOR.MINOR.PATCH) https://semver.org/

There is a lot of discussion around allowing or not breaking changes in minors when major is 0 but given we are doing it, I would stick to the pattern of: minor can have breaking changes, patch can't.

**Comment by karlbaker02** [comment]
README updated.

File: .version (hunk 1 above)
Comment (karlbaker02) [reply .version]: That's what I said - "It's a minor bump" :) Yes, given that this is a pre-release gem I think we are within limits to make a breaking change which only updates the minor, as you can't change the major without making it production-ready.

**Pull request merged**